src/components/calculator/SelectControl.tsx

```diff
@@ -36,7 +51,9 @@ export const GenericSelectControl: React.FunctionComponent<{
       <option value="">Select one...</option>
       {Object.keys(props.source).map((value, index) => (
         <option key={index} value={value}>
-          {props.source[value].label}
+          {props.source[value].label}{' '}
+          {!props.hiderisk &&
+            `[${showRiskMultiplier(props.source[value].multiplier)}]`}
```

Comment:
Can we throw this in an `<i>` tag for readability?

Reply:
You can't use other HTML tags inside an <option> tag. Which is why I opened #298

Reply:
Let me know if you have any other ideas.

Reply:
I'm ok punting this. <i> would have been nice if it was easy.